xclim/indices/run_length.py

```diff
@@ -1303,6 +1402,25 @@ def windowed_run_count_1d(arr: Sequence[bool], window: int) -> int:
     return np.where(v * rl >= window, rl, 0).sum()
 
 
+def windowed_max_run_sum_1d(arr: Sequence[float], window: int) -> float:
+    """Return the accumulated values of consecutive positive values in array for runs at least as long as given duration.
+
+    Parameters
+    ----------
+    arr : Sequence[float]
+        Input array (float).
+    window : int
+        Minimum duration of consecutive run to accumulate values.
+
+    Returns
+    -------
+    float
+        Accumulated values of positive values of a consecutive run at least `window` long.
+    """
+    # TODO: implement rse_1d function
+    raise NotImplementedError()
```

Comment:
Do we still use these functions in general? I think it's worth keep the 1d-function in case if we have specific performance issues, this might be useful to have alternative implementations, but in general, we don't necessarily need to build all new functions with a 1d version?

Reply:
Agree that this doesn't need a 1d function.

Reply:
Ok, in that case should I simply remove the corresponding parts again?

Reply:
Yes, I think removing the 1d stubs is ok.